nodejs/week1/src/backend/data/warmup.js

```diff
@@ -0,0 +1,40 @@
+
+//------------Create a class called Circle. It should have one property called radius.---------------------
+
+class circle { constructor(radius)
+
+       {  this.radius = radius;}
+  //----------------get diameter----------------------
+     getDiameter() 
+    {
+        return 2 * this.radius
+        
+    };
+    
+  //---------------get circumference-------------------
+
+    getCircumference ()
+    {
+        return 2 * Math.PI * this.radius;
+    };
+  //--------------- get Area ---------------------------
+
+     getArea ()
+     {
+         return Math.PI * this.radius * this.radius ;
+     };}
```

Comment:
here you can improve the code formatting a little bit, to make it more readible, are you using VS Code? there is a nice extension called 'prettier' that takes care of code formatting for you 👍

Reply:
thanks for the feedback i will fix the errors i have in my code.